pkg/install/v1alpha1/cstor_pool.go

```diff
@@ -308,6 +315,10 @@ spec:
               # OPENEBS_IO_CSTOR_ID env has UID of cStorPool CR.
             - name: OPENEBS_IO_CSTOR_ID
               value: {{ pluck .ListItems.currentRepeatResource .ListItems.nodeUidMap.nodeUid |first | splitList " " | first}}
+            {{- if ne $debugProfileVal "false" }}
+            - name: "OPENEBS_IO_DEBUG_PROFILE"
+              value: "{{- $debugProfileVal }}"
```

Comment:
I guess this is syntactically wrong. Valid values can be following:
```yaml
value: {{ $debugProfileVal | quote }}
```
I am not sure if quote is required!!